loadtest.py

```diff
@@ -69,19 +69,27 @@ def _run():
 async def test(session):
     storage = session.storage
 
+    # Respect the server limits.
+    _, config = await storage.get("/info/configuration")
+    print("Config: {}".format(json.dumps(config, indent=3)))
+    # fix up consts
+    payload = _PAYLOAD[:config.get("max_record_payload_bytes")]
+    # GET requests to meta/global
+    num_requests = min(get_num_requests('metaglobal'),
+                       config.get("max_post_records"))
+    batch_max_count = min(BATCH_MAX_COUNT, config.get("max_total_records"))
```

Comment:
```suggestion
    batch_max_count = min(_BATCH_MAX_COUNT, config.get("max_total_records"))
```

Reply:
i... how the hell did that pass locally?
thanks

Reply:
ah, because, apparently, i was running the script on a different machine. awesome.